src/web/app/pages-session/session-submission-page/session-submission-page.component.scss

```diff
@@ -4,3 +4,23 @@
   border: 0;
   box-shadow: 0 0 0 0;
 }
+
+.col-sm-3 {
+  text-align: center;
+}
+
+.col-sm-6 {
+  text-align: center;
+}
+
+$breakpoint-tablet: 577px;
```

Comment:
Is this really necessary?

Reply:
Hello! Thank you so much for reviewing our pull request. 
After thinking again, this portion of code can be removed, and we can write "min-width: 768px" (after changing sm to md) at the media query below and add "text-center" for each row. 
Am I on the right track? 
Thank you again for reviewing our PR!

Reply:
Yeap, looks okay to me

Reply:
We have modified our PR. Could you please review our modification? Thank you so much!